tinyagentos/tools/image_tool.py

```
@@ -29,24 +29,146 @@
                 "type": "integer",
                 "description": "Random seed for reproducibility (omit for random)",
             },
+            "model": {
+                "type": "string",
+                "description": "Pick from list_image_models output. When supplied, routes through the controller's scheduler which picks the right backend (NPU first, CPU fallback). Omit to call the default backend directly with lcm-dreamshaper-v7 (legacy path; bypasses the scheduler).",
+            },
+            "guidance_scale": {
+                "type": "number",
+                "description": "Classifier-free guidance scale. Higher values follow the prompt more strictly; 7.5 is balanced. Lower (1–4) for artistic flexibility, higher (10–15) for strict adherence.",
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)